index.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface IRandomWordsOptions {
```

Comment:
Pretty sure you only need this `index.d.ts` file in the root, and that'd be enough. 

Any time you add another param, just make sure it's added to this list. The `?` in the param means it's optional.

Reply:
Thank you for reviewing !